tests/db_functions/tests.py

```diff
@@ -426,6 +426,32 @@ def test_upper(self):
             lambda a: (a.upper_name, a.name)
         )
 
+    def test_trim(self):
+        Author.objects.create(name='  John ', alias='j')
+        Author.objects.create(name='Rhonda', alias='r')
+        authors = Author.objects.annotate(
+            lstrip=LStrip('name'),
+            ltrim=LTrim('name'),
+            rstrip=RStrip('name'),
+            rtrim=RTrim('name'),
+            strip=Strip('name'),
+            trim=Trim('name'),
+        )
+        self.assertQuerysetEqual(
+            authors.order_by('alias'), [
+                ('John ', 'John ', '  John', '  John', 'John', 'John'),
+                ('Rhonda', 'Rhonda', 'Rhonda', 'Rhonda', 'Rhonda', 'Rhonda'),
+            ],
+            lambda a: (a.lstrip, a.ltrim, a.rstrip, a.rtrim, a.strip, a.trim)
+        )
+        for function in (LStrip, LTrim, RStrip, RTrim, Strip, Trim):
+            with self.subTest(function=function):
+                with self.assertRaisesMessage(
+                    TypeError,
+                    "'%s' takes exactly 1 argument (2 given)" % function.__name__,
+                ):
+                    Author.objects.update(name=function('name', 'name'))
```

Comment:
Testing the error condition could be a separate test method and I don't think `Author.objects.update(name=` is needed to trigger the exception, is it?

Reply:
Yes, You're right :+1: